src/js/services/alerts.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('invertedIndex')
+  .service('alerts', function ($rootScope, $timeout) {
+
+    /**  configures the alert property
+     *
+     * @param {Strimg} message to alert
+     * @param {tye} type of message
+     * @param {Boolean} checks for falsy
+     * @param {Integer} time of display 
+     */
+
+    return function (message, type, show, timeout) {
+      // var alertTimeout;
+      $rootScope.alert = {
```

Comment:
Use `$rootScope` is not advisable [Don't Use $rootScope](https://www.quora.com/Is-it-preferable-to-use-rootscope-in-AngularJS-to-pass-data-from-one-controller-to-another) because this means that you are declaring a variable  into the global scope and that means anything can happen to your variable or whatever you put there. In this scenario, you can use `$q` promises in angular which will handle passing of data from services to the controller. Read more on [Promise Api](https://docs.angularjs.org/api/ng/service/$q)